server/routes/appointments/create-and-edit/editRoutes.ts

```diff
@@ -43,8 +44,25 @@ export default function Edit({ prisonService, activitiesService }: Services): Ro
   const startHandler = new StartJourneyRoutes(editAppointmentService)
   const locationRoutes = new LocationRoutes(activitiesService)
   const dateAndTimeRoutes = new DateAndTimeRoutes(activitiesService)
+  const confirmEditRoutes = new ConfirmEditRoutes(activitiesService, editAppointmentService)
   const applyToRoutes = new ApplyToRoutes(activitiesService, editAppointmentService)
 
+  // Cancel routes
+  const cancellationReasonRoutes = new CancellationReasonRoutes(editAppointmentService)
+
+  router.get(
+    '/start/cancel',
+    fetchAppointment(activitiesService),
+    fetchAppointmentOccurrence(activitiesService),
+    startHandler.CANCEL,
+  )
+  get('/cancel/reason', cancellationReasonRoutes.GET, true)
+  post('/cancel/reason', cancellationReasonRoutes.POST, CancellationReason)
+  get('/cancel/confirm', confirmEditRoutes.GET, true)
+  post('/cancel/confirm', confirmEditRoutes.POST, ConfirmEdit)
+  get('/cancel/apply-to', applyToRoutes.GET, true)
+  post('/cancel/apply-to', applyToRoutes.POST, ApplyTo)
```

Comment:
Not sure I have much of a preference either way, but I'm wondering if cancel is an edit route?

Reply:
I considered the same thing and ultimately went with edit due to the mode switching we have. I didn't want to add another one with more logic to test. Something to come back to when adding tests perhaps